README.rst

```diff
@@ -47,11 +46,13 @@ Dependencies
 
 scikit-learn requires:
 
-- Python (>= 3.6)
-- NumPy (>= 1.13.3)
-- SciPy (>= 0.19.1)
-- joblib (>= 0.11)
-- threadpoolctl (>= 2.0.0)
+- Python *(>= 3.6)*
```

Comment:
I find this reduces readability when not viewing as HTML

Reply:
no issue reverting